Description: Fix compat with sphinx 4
Author: Thomas Goirand <zigo@debian.org>
Origin: https://github.com/sqlalchemy/sqlalchemy/commit/c50183274728544e40e7da4fd35cf240da5df656
Last-Update: 2021-09-29

--- sqlalchemy-1.4.23+ds1.orig/lib/sqlalchemy/util/langhelpers.py
+++ sqlalchemy-1.4.23+ds1/lib/sqlalchemy/util/langhelpers.py
@@ -1453,7 +1453,7 @@ class hybridmethod(object):
     """Decorate a function as cls- or instance- level."""
 
     def __init__(self, func):
-        self.func = func
+        self.func = self.__func__ = func
         self.clslevel = func
 
     def __get__(self, instance, owner):
